source/pc/eathread_pc.cpp

```diff
@@ -504,7 +504,7 @@ void* EA::Thread::GetThreadStackBase()
 }
 
 
-#if defined(EA_PLATFORM_WIN32) && defined(EA_PROCESSOR_X86) && defined(_MSC_VER) && (_MSC_VER >= 1400)
+#if defined(EA_PLATFORM_WIN32) && defined(EA_PROCESSOR_X86) && defined(EA_COMPILER_MSVC) && (EA_COMPILER_MSVC >= 1400)
```

Comment:
use EA_COMPILER_VERSION?

Reply:
Ah, that one slipped past me, good catch.